@remirror/extension-yjs/src/yjs-extension.ts

```diff
@@ -0,0 +1,156 @@
+import {
+  CustomHandlerKeyList,
+  DefaultExtensionOptions,
+  HandlerKeyList,
+  PlainExtension,
+  StaticKeyList,
+  convertCommand,
+  invariant,
+} from '@remirror/core';
+import { redo, undo, yCursorPlugin, ySyncPlugin, yUndoPlugin } from 'y-prosemirror';
+import { Doc } from 'yjs';
+import { WebrtcProvider } from 'y-webrtc';
+import { Observable } from 'lib0/observable';
+import { css } from 'linaria';
+
+/**
+ * yjs typings are very rough; so we define here the interface that we require
```

Comment:
This is fixed in y-webrtc@10.1.5 - I forgot to publish the tsc files.

It might be a good idea to remove the Observable import from lib0. Just because of this import, this extension won't run in node<12 because lib0 is an esm module. The modules in the Yjs repo are not affected because their bundle points to `lib0/dist/[module].cjs` which is a commonjs module.

Reply:
Well.. let me know if it is not fixed :wink: Trying to keep the typescript definitions up to date.

Reply:
Awesome; Ifi was planning to contribute back some improved typings, but good to know they’re available soon! We need a “provider” type that all providers implement (webrtc, websockets), is there such a thing, @dmonad?

Reply:
That's a good idea. No there is not at the moment, but I'll add one in the next release https://github.com/yjs/yjs/issues/215

Reply:
Awesome :raised_hands: